README.md

```diff
@@ -77,7 +79,7 @@ Installing on macOS:
 
 - Install dependencies using Homebrew:
   ```
-  brew install autoconf automake libtool bdw-gc boost bison pkg-config
+  brew install autoconf automake libtool bdw-gc boost bison pkg-config protobuf
```

Comment:
Yes; I wasn't aware that any supported platforms were still shipping protobuf 2.x. I'll add some guidance to `README.md`.